bodhi/models/models.py

```diff
@@ -1763,7 +1763,7 @@ def days_to_stable(self):
             if comment.user.name == 'bodhi' and \
                 'can be pushed to stable now if the ' \
                 'maintainer wishes' in comment.text:
-                return (comment.timestamp - self.date_submitted).days
+                return (datetime.utcnow() - self.date_testing).days - self.release.mandatory_days_in_testing
```

Comment:
@lmacken @ralphbean Since the comment gets added just after `self.release.mandatory_days_in_testing` and we are considering `datetime.utcnow()` also, I have made  `(datetime.utcnow() - self.date_testing).days - self.release.mandatory_days_in_testing`. Otherwise it would return negative value.